xmtp/migrations/2023-07-10-223309_add_installations/up.sql

```diff
@@ -3,7 +3,6 @@ CREATE TABLE installations (
     user_address TEXT NOT NULL,
     first_seen_ns BIGINT NOT NULL,
     contact BLOB NOT NULL,
-    contact_hash TEXT NOT NULL,
```

Comment:
Contact hash doesn't really help at this moment. Removing. This is a breaking change as it was removed from the migration directly